src/_includes/dart-tool.md

```diff
@@ -0,0 +1,38 @@
+{{site.alert.note}}
+  As of Flutter's 1.19.0 dev release, the `dart` command
+  is now surfaced for you. Downloading the Flutter SDK
+  also downloads the compatible version of Dart,
+  but if you have downloaded the Dart SDK separately,
+  make sure that the Flutter version of `dart` is listed
+  first in your path, as the two versions may not be compatible.
+  The following command (on macOS, Linux, and Chrome OS),
+  tells you whether the `flutter` and `dart` commands
+  originate from the same `bin` directory and are therefore
+  compatible. (Some versions of Windows
+  support a similar `where` command.)
+
+  ```sh
+  $ which flutter; which dart
+  <path-to-flutter-sdk>/flutter-sdk/bin/flutter
+  /usr/local/bin/dart
+  ```
+
+  As shown above, the two commands are not coming from
+  the same `bin` directory. Update your path to use
+  commands from `<path-to-flutter-sdk>/bin` before it
+  uses commands from `/usr/local/bin` (in this case).
+  After updating your shell for the change to take effect,
+  run the command again, and you should now see:
+
+  ```sh
```

Comment:
It seems weird to make these `which` command sections copyable, since the output gets copied too. Can you turn off copying?

I thought it was weird that we use `sh` instead of `terminal` here, but the styling is atrocious when you have `<` or `;` in `terminal`. Oh well.

Reply:
I don't know how, but I'll look around.